SygicMapsKit/Classes/BrowseMap/SYMKBrowseMapViewController.swift

```diff
@@ -71,6 +73,24 @@ public class SYMKBrowseMapViewController: UIViewController {
         mapView.addMapMarkersCluster(mapSelectionManager.clusterLayer!)
     }
     
+    // MARK: - PoiDetail
+    
+    private func showPoiDetail(with poiDetailViewModel: SYMKPoiDetailDataSource) {
```

Comment:
to meno viewModel by som nahradil niecim inym

Reply:
lokalna variabla premenovana na poiDetailData typu SYMKPoiDetailDataSource